lib/linguist/languages.yml

```diff
@@ -4461,6 +4461,24 @@ SRecode Template:
   extensions:
   - ".srt"
   language_id: 335
+SSH Config:
+  type: data
+  group: INI
+  filenames:
+  - "ssh-config"
+  - "ssh_config"
+  - "sshconfig"
+  - "sshd-config"
+  - "sshd_config"
+  extensions:
+  - ".ssh_config"
+  - ".ssh-config"
+  - ".sshconfig"
+  - ".sshd-config"
+  - ".sshd_config"
```

Comment:
I couldn't find any file with the `.sshd-config` extension on GitHub, and only 61 files for the `.sshd_config` extension. Maybe we should only add new filenames in this pull request.

Reply:
I'm on board with adding just the filenames. Should I delete extensions altogether ?